src/backend/gl/src/device.rs

```diff
@@ -254,6 +256,27 @@ impl Device {
         Ok(())
     }
 
+    fn set_push_const_layout(
+        &self,
+        ast: &mut spirv::Ast<glsl::Target>,
+    ) -> Result<(), d::ShaderError> {
+        let resources = ast.get_shader_resources().map_err(gen_unexpected_error)?;
+
+        match self.share.info.shading_language.tuple() {
+            // Explicit uniform locations are only valid on OpenGL 4.3+
```

Comment:
What are we doing pre-4.3?

Reply:
Nothing because it's not supported... We could probably remove this since it's not actively being used.